src/main.rs

```diff
@@ -781,11 +781,13 @@ mod unit_tests {
                 file: None,
                 mode: ConsoleOutputMode::Null,
                 iommu: false,
+                unix_socket: None,
             },
             console: ConsoleConfig {
                 file: None,
                 mode: ConsoleOutputMode::Tty,
                 iommu: false,
+                unix_socket: None,
```

Comment:
Data structure should match command line to make API vs CLI easier.

Reply:
updated to keep it consistent